iso15118/secc/controller/simulator.py

```diff
@@ -413,6 +413,10 @@ async def get_energy_service_list(self) -> ServiceList:
 
         return service_list
 
+    async def is_external_authorization_done(self) -> bool:
+        """Overrides EVSEControllerInterface.is_external_authorization_done()."""
+        return True
```

Comment:
It would be better to leave the simulator at False as otherwise Contract wouldn't be offered by default in the simulator?